events/code_review_events/workflow.py

```diff
@@ -216,9 +257,140 @@ def publish_results(self, payload):
 
         return True
 
-    def parse_pulse(self, payload):
+    async def got_bugbug_test_select_end(self, payload):
+        bugbug_task_id = payload["status"]["taskId"]
+
+        # Get the Phabricator diff ID from bugbug task definition.
+        bugbug_task = self.community_tc["queue"].task(bugbug_task_id)
```

Comment:
Global try/except because of this stuff: it could fail for a million reasons

Reply:
Done.